debian.aws/config/OVERRIDES

```diff
@@ -0,0 +1,237 @@
+#
+# Disable various "net" modules which we don't use.
+#
+CONFIG_6LOWPAN=n
+CONFIG_6LOWPAN_NHC=n
+CONFIG_BT=n
+CONFIG_CAIF=n
+CONFIG_HAMRADIO=n
+CONFIG_IEEE802154=n
+CONFIG_NET_9P=n
+CONFIG_NET_DSA=n
+CONFIG_NFC=n
+CONFIG_RFKILL=n
+CONFIG_WIMAX=n
+CONFIG_WIRELESS=n
+
+#
+# Disable various "drivers" modules which we don't use.
+#
+CONFIG_ACCESSIBILITY=n
+CONFIG_ACPI=n
+CONFIG_ATA=n
+CONFIG_ATM_DRIVERS=n
+CONFIG_AUXDISPLAY=n
+CONFIG_BCMA=n
+CONFIG_CONNECTOR=n
+CONFIG_CRYPTO_HW=n
+CONFIG_DAX=n
+CONFIG_EISA=n
+CONFIG_EXTCON=n
+CONFIG_FPGA=n
+CONFIG_FSI=n
+CONFIG_GNSS=n
+CONFIG_GPIOLIB=n
+CONFIG_GREYBUS=n
+CONFIG_HSI=n
+CONFIG_I3C=n
+CONFIG_IDE=n
+CONFIG_IIO=n
+CONFIG_INFINIBAND=n
+CONFIG_INTERCONNECT=n
+CONFIG_IPACK_BUS=n
+CONFIG_LIBNVDIMM=n
+CONFIG_MAILBOX=n
+CONFIG_MCB=n
+CONFIG_MD=n
+CONFIG_MEDIA_SUPPORT=n
+CONFIG_MEMSTICK=n
+CONFIG_MMC=n
+CONFIG_MTD=n
+CONFIG_NEW_LEDS=n
+CONFIG_NTB=n
+CONFIG_NVM=n
+CONFIG_NVMEM=n
+CONFIG_OF=n
+CONFIG_PARPORT=n
+CONFIG_PARPORT_PANEL=n
+CONFIG_PCCARD=n
+CONFIG_PINCTRL=n
+CONFIG_PMIC_OPREGION=n
+CONFIG_PM_DEVFREQ=n
+CONFIG_POWERCAP=n
+CONFIG_PPS=n
+CONFIG_RAPIDIO=n
+CONFIG_REGULATOR=n
+CONFIG_RESET_CONTROLLER=n
+CONFIG_RTC_CLASS=n
+CONFIG_SFI=n
+CONFIG_SIOX=n
+CONFIG_SLIMBUS=n
+CONFIG_SOUNDWIRE=n
+CONFIG_SPI=n
+CONFIG_SPMI=n
+CONFIG_STAGING=n
+CONFIG_THERMAL=n
+CONFIG_THUNDERBOLT=n
+CONFIG_UIO=n
+CONFIG_USB_SUPPORT=n
+CONFIG_VFIO=n
+CONFIG_W1=n
+
+#
+# Disable various filesystem which we don't use.
+#
+CONFIG_9P_FS=n
+CONFIG_ADFS_FS=n
+CONFIG_AFFS_FS=n
+CONFIG_AFS_FS=n
+CONFIG_AUFS_FS=n
+CONFIG_BEFS_FS=n
+CONFIG_BFS_FS=n
+CONFIG_BTRFS_FS=n
+CONFIG_CEPH_FS=n
+CONFIG_CODA_FS=n
+CONFIG_ECRYPT_FS=n
+CONFIG_EFS_FS=n
+CONFIG_EROFS_FS=n
+CONFIG_EXFAT_FS=n
+CONFIG_EXT2_FS=n
+CONFIG_EXT3_FS=n
+CONFIG_EXT4_FS=n
```

Comment:
Are we sure we want to get rid of ext4? I know that we don't use it for anything but I wonder if there is ever any scenario that we need it as a backup.

Reply:
backup for what? I mean, we can included it.. but I'm not sure how/why we'd use it..

Reply:
I was curious if GRUB uses it or something for the recovery environment but I think that's ZFS all around too so I guess we are good without ext4. Dropping this.

Reply:
looks like docker uses an ext4 mountpoint.. so I'll add ext4 back in..
```
Oct 21 19:15:31 ip-10-110-251-69 check_docker_start_deps[15599]: mount: /var/lib/docker: unknown filesystem type 'ext4'.
Oct 21 19:15:31 ip-10-110-251-69 check_docker_start_deps[15403]: /opt/delphix/server/bin/check_docker_start_deps: failed to mount docker zvol
```

Reply:
ok glad we caught this!